src/Yandex/Allure/PHPUnit/Adapter/AllureAdapter.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Yandex\Allure\Adapter;
+namespace Yandex\Allure\PHPUnit\Adapter;
```

Comment:
I would like to have `Yandex\Allure\PHPUnit` namespace instead as well as class named `AllurePHPUnit` (or `PHPUnit` replaced with `PhpUnit` in both cases if it is applicable in PHP world)